Services.Test/StorageTest.cs

```diff
@@ -210,8 +211,44 @@ public StorageTest()
                         It.Is<string>(s => s == Storage.LOGO_KEY)),
                     Times.Once);
 
-            Assert.Equal(result.Image.ToString(), image);
-            Assert.Equal(result.Type.ToString(), type);
+            Assert.Equal(image, result.Image.ToString());
+            Assert.Equal(type, result.Type.ToString());
+            Assert.Null(result.Name);
+            Assert.False(result.IsDefault);
+        }
+
+        [Fact]
+        public async Task GetLogoAndNameAsyncTest()
```

Comment:
I'm trying to make sure that we follow this unit test naming pattern Should_ExpectedBehavior_When_StateUnderTest. It will make it more clear what the test is supposed to accomplish.